packages/dx-react-scheduler/docs/reference/toolbar.md

```diff
@@ -0,0 +1,57 @@
+# Toolbar Plugin Reference
+
+A plugin that renders the Scheduler's toolbar.
+
+## Import
+
+Use the following statement to import a plugin with embedded theme components:
+
+```js
+import { Toolbar } from '@devexpress/dx-react-scheduler-material-ui';
+```
+
+If you want to use custom components, you can import the themeless plugin:
+
+```js
+import { Toolbar } from '@devexpress/dx-react-scheduler';
+```
+
+## User Reference
+
+### Dependencies
+
+none
+
+### Properties
+
+Name | Type | Default | Description
+-----|------|---------|------------
+rootComponent | ComponentType&lt;[Toolbar.RootProps](#toolbarrootprops)&gt; | | A component that renders the toolbar's root element.
+flexibleSpaceComponent | ComponentType&lt;[Toolbar.FlexibleSpaceProps](#toolbarflexiblespaceprops)&gt; | | A component that renders the toolbar's empty area.
+
+## Interfaces
+
+### Toolbar.RootProps
+
+Describes properties passed to a component that renders the toolbar's root element.
+
+Field | Type | Description
+------|------|------------
+children? | ReactNode | A React node to be placed in the toolbar.
```

Comment:
A React node that should be placed in the toolbar?

Reply:
Yes